doc/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>CARTO.js</title>
+  <link type="text/css" rel="stylesheet" href="docs/v4/styles/jsdoc-default.css">
+</head>
+<body>
+  <h1 class="page-title">CARTO.js <span style="color:#0095dd;">v4</span></h1>
+  <ul style="margin:24px;">
+    <li><h3><a href="docs/v4">API docs</a></h3></li>
```

Comment:
should this be doc/v4?

Reply:
This file is the main index.html in gh-pages: https://github.com/CartoDB/cartodb.js/blob/gh-pages/index.html. It is copied with the "docs" directory. I think we could move it to another directory, may be "config/docs"